optimum/quanto/models/transformers_models.py

```diff
@@ -52,10 +53,14 @@ def __getattr__(self, name: str) -> Any:
         except AttributeError:
             wrapped = self.__dict__["_wrapped"]
             return getattr(wrapped, name)
-
-    def forward(self, *args, **kwargs):
+    
+    def __call__(self, *args, **kwargs):
         return self._wrapped.forward(*args, **kwargs)
+      
+    def forward(self, *args, **kwargs):
+        return self.model.forward(*args, **kwargs)
```

Comment:
This reverts the change I just made to fix a bug ...

Reply:
Sorry, I will change it.